etc/config/javascript.amazon.properties

```diff
@@ -0,0 +1,16 @@
+compilers=javascript
+defaultCompiler=javascript
+
+supportsBinary=false
+interpreted=true
+compilerType=v8
+objdumper=
+demangler=
+postProcess=
+options=
+supportsExecute=false
+stubText=function add(a, b) { return a + b; } for (var i = 0; i < 10; i++) { add(i, 1) }
```

Comment:
I _think_ this is unused: In C++ apps this is the extra text we add to generate a `main(){}` function to let us link a binary, if the user hasn't specified a main of their own. I'm pretty sure this is unused here

Reply:
Yes I added it when I copied a config file but it didn't really have an effect. I am removing it.